cobalt/cache/cache.cc

```diff
@@ -25,41 +25,41 @@
 #include "base/strings/string_number_conversions.h"
 #include "base/values.h"
 #include "cobalt/configuration/configuration.h"
-#include "net/disk_cache/cobalt/cobalt_backend_impl.h"
+#include "cobalt/network/disk_cache/cobalt_backend_impl.h"
 #include "starboard/configuration_constants.h"
 #include "starboard/extension/javascript_cache.h"
 #include "starboard/system.h"
 
 namespace {
```

Comment:
This is unrelated to the change, but do you know why this unnamed namespace is outside of the named namespaces below? I feel like in general I see it the other way around, and that would give you a slightly easier time referencing members under the cobalt::network namespace, but I don't see explicit style recommendations for doing it one way vs the other